python/gui/auto_generated/editorwidgets/qgsrelationwidgetwrapper.sip.in

```diff
@@ -113,6 +113,39 @@ Returns the buttons which are shown
 .. versionadded:: 3.16
 %End
 
+    bool forceSuppressFormPopup() const;
+%Docstring
+Determines the force suppress form popup status that is configured for this widget
+
+.. versionadded:: 3.16
+%End
+
+    void setForceSuppressFormPopup( bool forceSuppressFormPopup );
+%Docstring
+Sets force suppress form popup status to ``forceSuppressFormPopup`` for this widget
+and for the vectorLayerTools (if true).
+This flag will override the layer and general settings regarding the automatic
+opening of the attribute form dialog when digitizing is completed.
+
+.. versionadded:: 3.16
+%End
+
+    QVariant nmRelationId() const;
+%Docstring
+Determines the nmRelationId used for the many-to-many cardinality.
```

Comment:
```suggestion
Determines the relation id of the second relation involved. Used for many-to-many cardinality.
```

@signedav does this help readability?

Reply:
Without the "Used for many-to-many cardinality." part...